include/deal.II/base/work_stream.h

```diff
@@ -699,26 +696,36 @@ namespace WorkStream
       {
         // create the three stages of the pipeline
         IteratorRangeToItemStream<Iterator, ScratchData, CopyData>
-          iterator_range_to_item_stream(begin,
+             iterator_range_to_item_stream(begin,
                                         end,
```

Comment:
This indent looks odd. I guess the `indent` worker will complain in a second.

Reply:
Nope. Don't ask me why, but this is how clang-indent likes it.